packages/core/admin/admin/src/pages/Admin/index.js

```diff
@@ -87,6 +88,7 @@ const Admin = () => {
               <InstalledPluginsPage />
             </Route>
             <Route path="/404" component={NotFoundPage} />
+            <Route path="/500" component={InternalErrorPage} />
```

Comment:
ℹ️  We init the 500 page although there is no redirection to /500 in the app for the moment

Reply:
Yes that's a good idea!

I think that at some point we should improve the error handling in the admin by having two different views:
- one in production that redirects the user to the 500 page
-  one in development where we implement an error boundary properly


Anyway good work!